charts/hono/values.yaml

```diff
@@ -1718,15 +1715,31 @@ kafka:
     # Note that this could require creating RBAC rules, for more information refer to
     # https://github.com/bitnami/charts/tree/master/bitnami/kafka#accessing-kafka-brokers-from-outside-the-cluster
     create: true
+  # limit Kafka JVM's memory consumption
+  heapOpts: "-Xms620M -Xmx620M"
+  resources:
+    requests:
+      memory: "800Mi"
+    limits:
+      memory: "800Mi"
+
   zookeeper:
     persistence:
       enabled: false
     auth:
-      enabled: true
-      clientUser: "zookeeperUser"
-      clientPassword: "zookeeperPassword"
-      serverUsers: "zookeeperUser"
-      serverPasswords: "zookeeperPassword"
+      client:
+        enabled: true
+        clientUser: "zookeeperUser"
+        clientPassword: "zookeeperPassword"
+        serverUsers: "zookeeperUser"
+        serverPasswords: "zookeeperPassword"
+    # limit Zookeeper JVM's memory consumption
+    heapSize: 400
+    resources:
+      requests:
+        memory: "500Mi"
```

Comment:
How about keeping the zookeeper default of `256Mi` here for the `requests` value? I would also consider reducing the corresponding kafka resources/requests/memory configuration. Both would reduce the overall minimal memory requirements which could for example be relevant when installing the cloud2edge chart locally for testing purposes.

Reply:
I am not sure if we can reduce the Java heap size of the Kafka broker even further. The default is 1024M and based on my experience with trying to reduce the heap size when running the ITs locally, I do not believe that we can go down much more.
We might be able to configure a smaller amount of requested memory but IMHO the process(es) will very quickly try to claim more than the initial amount of memory so in the end I do not think that this will help with running at a reduced memory footprint. Have you made other experiences?

Reply:
In my tests, the Kafka pod took 570MiB minimum (having sent some messages for 1 device). So, I agree, there's not much room for reducing the requested memory size (I guess using 650MiB would also work, but then there's not much of a difference to the 800MiB size).
For Zookeeper though, I only saw usage of 130MiB, so I guess using 256MiB as requested size seems feasible there.